src/mumble/ServerHandler.cpp

```diff
@@ -774,9 +774,10 @@ void ServerHandler::serverConnectionConnected() {
 	MumbleProto::Version mpv;
 	mpv.set_release(u8(QLatin1String(MUMBLE_RELEASE)));
 
-	unsigned int version = Version::getRaw();
-	if (version) {
-		mpv.set_version(version);
+	Version::mumble_raw_version_t version = Version::getRaw();
+	if (version != Version::UNKNOWN) {
+		mpv.set_version_2(version);
+		mpv.set_version(Version::toLegacyVersion(version));
```

Comment:
Maybe it makes sense to write a wrapper function that one is supposed to use instead of setting these fields manually? Otherwise, it seems too easy to e.g. forget to always also set the legacy version field